book/src/volume-expansion.md

```diff
@@ -24,7 +31,11 @@ For details,  see the [CSI spec](https://github.com/container-storage-interface/
 ## Deploying volume expansion functionality
 
 The Kubernetes CSI development team maintains [external-resizer](external-resizer.md) Kubernetes CSI [Sidecar Containers](sidecar-containers.md).
-This sidecar container implements the logic for watching the Kubernetes API for Persistent Volume claim edits and issuing the appropriate CSI expansion calls against a CSI endpoint. For more details, see [external-resizer](external-resizer.md).
+This sidecar container implements the logic for watching the Kubernetes API for Persistent Volume claim edits and issuing `ControllerExpandVolume` RPC call against a CSI endpoint and updating `PersistentVolume` object to reflect new size.
```

Comment:
Is the sidecar needed for NodeExpand? Can you clarify here

Reply:
fixed